server/index.js

```diff
@@ -31,6 +32,13 @@ const createApp = () => {
   // compression middleware
   app.use(compression());
 
+  // parcel middleware
+  const entryFile = path.join(__dirname, "..", "client", "index.html");
+  const bundler = new Bundler(entryFile, {
+    outDir: "./dist"
+  });
+  app.use(bundler.middleware());
```

Comment:
This is the "big deal". Rather than run a side-process, we can attach parcel into our server.

Sort of like the static file middleware, parcel watches for requests and if it's a match for the bundle or `index.html`, it generates the bundle just-in-time for that request.

Never get bitten by a stale bundle again.